public/stylesheets/itvc.css

```diff
@@ -1665,3 +1665,23 @@ margin-bottom: 0 !important;
     margin-top: 30px;
 }
 
+.what-you-owe-total {
+    width: 990px;
+    height: 110px
+}
+.what-you-owe-total-heading {
+    font-size: 24px !important;
+    line-height: 1.25 !important;
+    margin-bottom: 20px;
+    margin-top: 0px;
+}
+
+.what-you-owe-total-values {
+    font-size: 36px !important;
+    line-height: 1.11 !important;
+    font-weight: bold;
+}
+
+.what-you-owe-total-values-bold {
+    font-weight: 700 !important;
+}
```

Comment:
Just a note: these styles probably won't be required after migration to the new frontend system. Or probably they'll be replicated in new application.css during migration of the page.

Reply:
Yes agree, will let Jack who is working on migration know about it.